main.tex

```diff
@@ -361,9 +328,9 @@ \section{Astropy Core Package v2.0}
 packages to utilize for a common interface (e.g., \texttt{NDData}).
 In this section, we highlight new features introduced or substantially improved
 since version v0.2 (previously described in \citealt{astropy}).  The \astropypkg
-provides a full log of changes \footnote{https://github.com/astropy/astropy/blob/master/CHANGES.rst}
+provides a full log of changes \footnote{\url{https://github.com/astropy/astropy/blob/master/CHANGES.rst}}
```

Comment:
shouldn't this be "stable" rather than master?

Reply:
switching to stable would be fine with me -- and more appropriate for the v2.0 release